docs/ref/http_scaled_object.md

```diff
@@ -0,0 +1,33 @@
+# The `HTTPScaledObject`
+
+Each `HTTPScaledObject` looks approximately like the below:
+
+```yaml
+kind: HTTPScaledObject
+apiVersion: http.keda.sh/v1alpha1
+metadata:
+    name: xkcd
+spec:
```

Comment:
As a user, I'd be wondering how I configure when it should scale?

Reply:
Not sure what you mean. Is there another time it should scale besides when HTTP requests come into (or stop coming into) the system?

Reply:
I see what you mean now. We do 0->n scaling by default and you're right, there's no way to change that at the moment. Would you mind creating an issue to add that configurability?

Reply:
I would most definately add some more clear docs around this then, because I had the impression we supported `0->n` scaling and was confused about defining when to scale.

Reply:
https://github.com/kedacore/http-add-on/issues/101